applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_static_shifted_boundary_solver.py

```diff
@@ -0,0 +1,67 @@
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# Import applications
+import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
+
+# Import base class file
+from KratosMultiphysics.StructuralMechanicsApplication import structural_mechanics_static_solver
+
+def CreateSolver(main_model_part, custom_settings):
+    return StaticMechanicalShiftedBoundarySolver(main_model_part, custom_settings)
+
+class StaticMechanicalShiftedBoundarySolver(structural_mechanics_static_solver.StaticMechanicalSolver):
+
+    def __init__(self, main_model_part, custom_settings):
+        # Construct the base solver and validate the remaining settings in the base class
+        super().__init__(main_model_part, custom_settings)
+
+        # Overwrite the base solver minimum buffer size
+        self.min_buffer_size = 1
+
+        KratosMultiphysics.Logger.PrintInfo(self.__class__.__name__, "Construction finished")
+
+    @classmethod
+    def GetDefaultParameters(cls):
+        this_defaults = KratosMultiphysics.Parameters(r"""{
+            "conforming_basis" : true,
+            "extension_operator_type" : "MLS",
```

Comment:
```suggestion
            "extension_operator_type" : "mls",
```

Reply:
I put it in capital letters as it is an acronym. Note that changing it to lower case would imply changing the `ShiftedBoundaryMeshlessInterfaceUtility` settings as well.

Reply:
i still recommend to change it, so far we do it for most acronyms too
Up to you ;)

Reply:
Fair enough. If we're changing it, I'll do it at once (note that this involves another PR as well as things that are already merged to the `master`).

Reply:
@philbucher if this makes sense for you I'll proceed with the merge.

Reply:
this one is ok for me, but I would still add the check regarding the geometry (see my other comment)

Reply:
Done.